amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -998,11 +998,91 @@ type Profile @model {
   Metadata associated with the user's profile
   """
   meta: ProfileMetadata
-
   """
   The user associated with this profile
   """
   user: User! @belongsTo(fields: ["id"])
+  """
+  List of a user's notifications
+  """
+  notifications: [Notification] @hasMany
+  """
+  Notification settings
+  """
+  notificationSettings: NotificationSettings
+}
```

Comment:
What's the reasoning behind putting notifications on the profile model while the notification model itself has a relationship with the user model?

Reply:
None this is an oversight on my part which was left over from a previous experimentation where I had attached the Notifications to the user model.

Over time as I worked on this feature I decided that the `Profile` model was the correct place for Notifications as it uses elements of that model (such as the user's email) and generally seems to be where we store information about the user. Happy to be argued against however :)

I've updated the model to change the notification model to have a relationship with the *profile* now.